### PR TITLE
Fix BSBLan documentation

### PR DESCRIPTION
**Pull request by liudger**
Link to the heating systems, and updated the documentation to corresponded to the latest development of bsblan board

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by frenck** [comment]
There are all kinds of unrelated commits in this PR, could you please git rebase it onto the upstream `current` branch?

Thanks! 👍

**Comment by liudger** [comment]
> There are all kinds of unrelated commits in this PR, could you please git rebase it onto the upstream `current` branch?
> 
> Thanks! 👍

Made an error. Going to rebase now

File: source/_integrations/bsblan.markdown [hunk 1]
```diff
@@ -38,7 +38,11 @@ After completing the configuration flow, the BSBLan Climate integration will be
 available.
 
 For more documentation of the BSBLan device, check the [manual](https://1coderookie.github.io/BSB-LPB-LAN_EN/).
-The device support quite a few controllers from different manufacturers.
-[list of supported models](https://1coderookie.github.io/BSB-LPB-LAN_EN/chap03.html#313-other-manufacturers)
 
-The integration is tested with firmware latest stable version `v0.43`.
+**_To see a more detailed listing of the reported systems which are successfully used with BSB-LAN please follow the corresponding link:_**
+
+- **[`Brötje`](https://1coderookie.github.io/BSB-LPB-LAN_EN/chap03.html#311-broetje)**
```
Comment (klaasnicolaas) [inline source/_integrations/bsblan.markdown]: I think it's not necessary to make this bold

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/bsblan.markdown (hunk 1 above)
Comment (liudger) [reply source/_integrations/bsblan.markdown]: indeed

Comment (frenck) [reply source/_integrations/bsblan.markdown]: @liudger That includes the list of authors 😉

Comment (liudger) [reply source/_integrations/bsblan.markdown]: done 😊

**Review by frenck — APPROVED** [review]
Thanks, @liudger 👍

**Pull request merged**